packages/contentstack-audit/test/unit/commands/fix.test.ts

```diff
@@ -12,19 +13,17 @@ describe('AuditFix command', () => {
   } as FileTransportInstance;
 
   describe('AuditFix run method', () => {
-    const test = fancy.loadConfig({ root: process.cwd() });
-    test
-      .stdout({ print: process.env.PRINT === 'true' || false })
-      .stub(fs, 'rmSync', () => {})
-      .stub(winston.transports, 'File', () => fsTransport)
-      .stub(winston, 'createLogger', () => ({ log: () => {}, error: () => {} }))
-      .stub(AuditBaseCommand.prototype, 'start', () => {})
-      .spy(AuditBaseCommand.prototype, 'start')
-      .command(['cm:stacks:audit:fix'])
-      .it('should trigger AuditBaseCommand start method', ({ stdout, spy }) => {
-        expect(stdout).to.be.empty.string;
-        expect(spy.start.callCount).to.be.equals(1);
-        expect(spy.start.args).deep.equal([['cm:stacks:audit:fix']]);
-      });
+    sinon.stub(fs, 'rmSync').callsFake(() => {});
+    sinon.stub(winston.transports, 'File').callsFake(() => fsTransport);
+    sinon.stub(winston, 'createLogger').call(() => ({ log: () => {}, error: () => {} }));
+    const spy = sinon.stub(AuditBaseCommand.prototype, 'start').callsFake(() => {
+      return Promise.resolve(true);
+    });
```

Comment:
@cs-raj , try to use startSpy name instead of spy

Reply:
Done